draft-ietf-tls-tls13.md

```diff
@@ -3470,11 +3499,12 @@ certificate_list
 extensions:
 : A set of extension values for the CertificateEntry. The "Extension"
   format is defined in {{extensions}}. Valid extensions for server certificates
-  include OCSP Status extension ({{RFC6066}}) and
-  SignedCertificateTimestamps ({{!RFC6962}}). Extensions in the Certificate
-  message from the server MUST correspond to one from the ClientHello message.
-  Extensions in the Certificate from the client MUST correspond with an
-  extension in the CertificateRequest message from the server.
+  at present include OCSP Status extension ({{RFC6066}}) and
+  SignedCertificateTimestamps ({{!RFC6962}}) and future extensions may
+  be defined for this message as well. Extensions in the Certificate
```

Comment:
I'm not sure this is needed, but it's ungrammatical in any case.

I think you instead need a semicolon "; future extensions may"